miscellaneous/build_templates/analysis.yml

```diff
@@ -68,6 +69,37 @@ jobs:
         displayName: 'Build (Python Library)'
         condition: and(eq('${{ parameters.language }}', 'python'), eq(${{ parameters.lib }}, true))
 
+      # - Jupyter
+      - script: |
+          echo Requirements
+          call pip install -r requirements.txt
+          echo TestRequirements
+          call pip install -r test-requirements.txt
+          echo ConvertToPython
+          jupyter nbconvert --to script *.ipynb
+          echo Complete
+        workingDirectory: $(Build.SourcesDirectory)$(projPath)/
+        displayName: 'Build (Jupyter): Install dependencies'
+        condition: eq('${{ parameters.language }}', 'jupyter')
+
+      # - Angular
```

Comment:
Looks like you might be accidentally adding the Angular build back, I think this would break the Angular samples.